src/token/history/history.resolver.ts

```diff
@@ -1,33 +1,26 @@
 import { Args, Mutation, Resolver } from '@nestjs/graphql'
 import { Prisma } from '@prisma/client'
 import { GetCurrentUserId } from '../../auth/decorators/get-current-user-id.decorator'
-import { Roles } from '../../auth/decorators/roles.decorator'
 import { TokenHistory } from '../../token/history/entities/token-history.entity'
 import { TokenHistoryService } from '../../token/history/history.service'
+import { CreateHistoryDto } from './dto/create-history.input'
 
 @Resolver(() => TokenHistory)
 export class TokenResolver {
     constructor(private readonly historyTokenService: TokenHistoryService) {}
 
     @Mutation(() => TokenHistory, { nullable: true })
-    async getMyTokenHistoryById(@GetCurrentUserId() userId: number): Promise<TokenHistory> {
-        const tokenHistoryId= await this.historyTokenService.getMyTokenHistoryById(userId)
-        if (!tokenHistoryId) {
-            throw new Error('token history not found')
-        }
-
-        return tokenHistoryId
+    async getMyTokenHistory(@GetCurrentUserId() input: CreateHistoryDto): Promise<TokenHistory[]> {
```

Comment:
method ``@GetCurrentUserId()`` return userId so you must use this code ``@GetCurrentUserId() userId: number``

Why you use ``CreateHistoryDto`` when you select data ? Do be attentive!